main.go

```diff
@@ -46,11 +47,11 @@ func setupApp() *App {
 
 	c.Version = Version
 
-	c.DNSProcPath, err = getProcPath()
-	if err != nil {
-		log.Fatal(err)
-	}
-	c.DNSProcPath = path.Join(c.DNSProcPath, "hnsd")
+	// c.DNSProcPath, err = getProcPath()
+	// if err != nil {
+	// 	log.Fatal(err)
+	// }
+	// c.DNSProcPath = path.Join(c.DNSProcPath, "hnsd")
```

Comment:
actually this is useful, it gives the path to hnsd. Can use this instead of hardcoded path below